libraries/vdf/src/sortition.cpp

```diff
@@ -23,7 +22,7 @@ uint16_t VdfSortition::calculateDifficulty(SortitionParams const& config) const
     if (isStale(config)) {
       difficulty = config.vdf.difficulty_stale;
     } else {
-      difficulty = config.vdf.difficulty_min + threshold_ % (config.vdf.difficulty_max - config.vdf.difficulty_min);
+      difficulty = config.vdf.difficulty_min + threshold_ % (config.vdf.difficulty_max - config.vdf.difficulty_min + 1);
```

Comment:
Why need "+1" here? Why not increase difficulty_max to 22?

Reply:
it was incorrect before. Imagine the range is between 5 and 10. if you do"5 + X % (10 - 5) the values you get are 5, 6, 7, 8 and 9, you need +1 to actually get the 10 as well.